tableauserverclient/server/endpoint/schedules_endpoint.py

```diff
@@ -84,5 +89,4 @@ def add_to(resource, type_, req_factory):
             items.append((datasource, "datasource", RequestFactory.Schedule.add_datasource_req))
 
         results = (add_to(*x) for x in items)
-
-        return all(results)
+        return filter(lambda x: not x.result, results)
```

Comment:
Just curious -- why do we have to filter out empty results... or maybe the question is "why does Server return empty results?"